storehaus-memcache/src/main/scala/com/twitter/storehaus/memcache/MemcacheStore.scala

```diff
@@ -119,23 +120,27 @@ class MemcacheStore(val client: Client, val ttl: Duration, val flag: Int)
 {
   override def withPutTtl(ttl: Duration) = new MemcacheStore(client, ttl, flag)
 
-  override def get(k: String): Future[Option[ChannelBuffer]] = client.get(k)
+  override def get(k: String): Future[Option[ChannelBuffer]] =
+    client.get(k).map(_.map(ChannelBufferBuf.Owned.extract))
```

Comment:
Would you like if I pattern matched on the Option here to avoid the allocation for None case (using Future.None)?